Odata-docs/client/grouping-and-aggregation.md

```diff
@@ -0,0 +1,481 @@
+---
+title: "Grouping and Aggregation in OData Client"
+description: "This tutorial describes how to group and aggregate data in Odata Client"
+
+author: gathogojr
+ms.author: jogathog
+ms.date: 7/25/2022
+ms.topic: article
+ 
+---
+# Grouping and Aggregation in OData Client
+**Applies To**: [!INCLUDE[appliesto-odataclient](../includes/appliesto-odataclient-v7.12.1.md)]
+
+OData Client [v7.12.1](https://www.nuget.org/packages/Microsoft.OData.Client/) introduced support for grouping and aggregation. In this page, we look at the various supported scenarios. The sample data from the 
+[OASIS OData Extension for Data Aggregation Version 4.0](http://docs.oasis-open.org/odata/odata-data-aggregation-ext/v4.0/cs01/odata-data-aggregation-ext-v4.0-cs01.html#_Toc378326287) will be used to illustrate the scenarios supported by this feature.
```

Comment:
Hopefully the linked data never changes :).

Also, the image in the data example seems low-res from my end, it's hard to read.

Reply:
@habbes I have added the image to assets and also made the link point to a more crisp version of the image